docs/pages/versions/unversioned/distribution/app-signing.md

```diff
@@ -2,10 +2,49 @@
 title: App signing
 ---
 
-Process is automated for iOS and Android, but in both cases you can choose to provide your own overrides. Both `expo build:ios` and
+Expo automates the process of signing your app for iOS and Android, but in both cases you can choose to provide your own overrides. Both `expo build:ios` and
 `expo build:android` commands generate signed applications ready to be uploaded into respective stores.
 
-## Android 
+On this page, we'll talk about the credentials that each platform requires. If you're curious about how we store your credentials on our end, take a look at our [security documentation](../security/).
+
+## iOS
+
+The 3 primary iOS credentials, all of which are associated with your Apple Developer account, are:
+
+- Distribution Certificate
+- Provisioning Profiles
+- Push Notification Keys
+
+Whether you let Expo handle all your credentials, or you handle them all yourself, it's smart to understand what each of these credentials mean, when and where they're used, and what happens when they expire or are revoked.
```

Comment:
```suggestion
Whether you let Expo handle all your credentials, or you handle them all yourself, it can be valuable to understand what each of these credentials mean, when and where they're used, and what happens when they expire or are revoked.
```